web/src/components/navbar/NavBar.tsx

```diff
@@ -37,8 +37,21 @@ export const NavBar = (props: Props): ReactElement => {
 
   if (props.logoOnly) {
     return <NavBarLogoOnly />;
-  } else if (props.landingPage) {
+  } else if (props.landingPage && isLargeScreen) {
     return <NavBarLanding />;
+  } else if (props.landingPage && !isLargeScreen) {
+    return (
+      <>
+        <NavBarMobile
+          scrolled={scrolled}
+          task={props.task}
+          showSidebar={props.showSidebar}
+          hideMiniRoadmap={props.hideMiniRoadmap}
+          landing
+        />
+        <div className={!isLargeScreen && scrolled ? "padding-top-6" : ""} />
+      </>
+    );
```

Comment:
With the additional `isLargeScreen` check for the previous condition, this case should fall to the final else. This is redundant. You can kill this.

Reply:
I can't write a comment on line 66 but `!isLargeScreen` check in className can be removed because we're checking that before that ternary is evaluated.

```
className={scrolled ? "padding-top-6" : ""}
```